Bakis/ClientApp/src/app/components/base-sight/base-sight.component.html

```diff
@@ -21,9 +21,15 @@
           </mat-form-field>
           <mat-form-field class="full-width">
             <input matInput type="number" formControlName="latitude" placeholder="Latitude">
+            <button mat-button matSuffix mat-icon-button (click)="openMapModal()">
```

Comment:
as far as I understand this requires you to both manually set lat/lang is there a way to have them both set from map at the same time?